deploy/kubernetes/elastic-agent-kustomize/README.md

```diff
@@ -0,0 +1,14 @@
+# Kustomize Kubernetes templates examples
+
+## Getting started
+
+This is the list of [kustomize](https://github.com/kubernetes-sigs/kustomize) templates provided in order to install  
```

Comment:
This sentence does not make a lot of sense?

Can we also provide a nice overview of how many approaches we support and a bit of description? Ideally the user that lands in this README should be able to quickly understand what we support and navigate easily to the specific section that is interested into.